config/config.yaml

```diff
@@ -85,6 +85,12 @@ calling:
   freebayes:
     activate: true
   # See https://varlociraptor.github.io/docs/calling/#generic-variant-calling
+  ScanITD: 
+    activate: false
+    tmpDIR: "/tmp/ScanITD"  # set directory for temporary files
```

Comment:
This should be hidden from the user. jUST USE THE `temp()` functionality of Snakemake.
```suggestion
```